### PR TITLE
New developer admins

### PR DESCRIPTION
**Pull request by ghost**
An update to the recently added developers.

### CODE REVIEW

**Comment by JeromSar** [comment]
Thanks for contributing. Your PR will be merged whenever possible.

**Comment by Commodore64x** [comment]
Wild must be removed from this list, he has retired.

**Review by ghost — CHANGES_REQUESTED** [review]
Wild must be removed from this list, he has retired.

**Comment by mayokek** [comment]
Doesn't mean he has to be removed.

**Comment by ghost** [comment]
@Commodore64x, he's in the same situation as Madgeek. He's indefinitely inactive until he returns. It'll be useless to remove him as if he decides to come back we will have to re-compile. He didn't retire, just invactive.

**Comment by ghost** [comment]
@JeromSar, there aren't any issues here as I'm aware. This can me merged whenever.

**Comment by GracieWargo** [comment]
Or you could just add a dev string to the config.yml. Duuuhhhh much easier so you don't have to recompile whenever you add a dev or remove

**Comment by Wild1145** [comment]
@GracieWargo The development system within TFM is designed for TFM developers, not server developers. So there is no reason to move it out of a place whereby developers will need to change it, not server owners / those with no experience.

**Review by Wild1145 — APPROVED** [review]
TravisCI Builds passing, all looks good to me https://travis-ci.org/Wild1145/TotalFreedomMod/builds/237814066?utm_source=github_status&utm_medium=notification

**Review by Wild1145 — DISMISSED** [review]
There are now merge conflicts with the base branch. Please resolve these before they can be merged.

**Comment by robingall2910** [comment]
The actual change has been done in Madgeek's commit 85fe41519647bcd508444e2d3e782a70c53e7a23

**Review by Wild1145 — APPROVED** [review]

**Review by lemonsked — APPROVED** [review]

File: src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
     //
     public static final String SAVED_FLAGS_FILENAME = "savedflags.dat";
     // See https://github.com/TotalFreedom/License - None of the listed names may be removed.
-    public static final List<String> DEVELOPERS = Arrays.asList("Madgeek1450", "Prozza", "Wild1145", "WickedGamingUK", "aggelosQQ");
+    public static final List<String> DEVELOPERS = Arrays.asList("Madgeek1450", "Prozza", "Wild1145", "WickedGamingUK", "aggelosQQ", "marcocorriero", "Commodore64x");
```
Comment (lemonsked) [inline src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java:39]: Change WickedGamingUK's name to Fyrsta, as he has changed his name,
https://namemc.com/profile/Fyrsta.2

**Review by lemonsked — CHANGES_REQUESTED** [review]

File: src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java (hunk 1 above)
Comment (Wild1145) [reply src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java:39]: Good catch, I haddnt realised he'd changed his in game name.

**Comment by ghost** [comment]
@Wild1145 This has been superseeded by #2059

**Comment by mayokek** [comment]
@Commodore64x, a PR cannot be duplicated.

**Comment by Wild1145** [comment]
@Commodore64x Given this is ready to merge (more or less) and the other still has a number of changes, we would be better off reverting these changes in #2059 and only adding them here.

File: src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java (hunk 1 above)
Comment (ghost) [reply src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java:39]: Wild isn't actually a dev anymore so his name should be removed.

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java (hunk 1 above)
Comment (Wild1145) [reply src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java:39]: Incorrect. As I've stated countless times on other discussions, this is a list of TFM developers by project, not TF Developers.

**Comment by JeromSar** [comment]
The host repository was deleted.

**Pull request closed without merging**